### PR TITLE
Added hacktoberfest events scrapper

### LINKED ISSUE #22: Scrape Hacktoberfest Events
# Description

Script to scrape the hacktoberfest events for 2020 and store them in a csv file.

## Type of issue

- [x] Feature (New Script)

## Checklist:

- [x] I have read the project guidelines.
- [x] I have checked all the existing projects, before submitting a new project issue.
- [x] I have checked previous issues to avoid duplicates.
- [x] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by AdityaJ7**
# Description

Script to scrape the events related to hacktoberfest 2020.

Fixes #22

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [x] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [x] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Review by tusharnankani — APPROVED** [review]
The script works well! 👍🏻

File: Scripts/Web_Scrappers/Hacktoberfest_Events/hacktoberfest_events.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    events = {}
+    for i, tablerow in enumerate(tablerows):
+        location = tablerow.find("td", {"class": "location"}).text
+        link = tablerow.find("a")['href']
+        name = tablerow.find("td", {"class": "event_name"}).text.strip()
+        date = tablerow.find("td", {"class": "date is-hidden"}).text.strip()
+        events[i] = [name, date, location, link]
+    df1 = pd.DataFrame.from_dict(events, orient='index')
+    df1.columns = ['Name', 'Date', 'Location', 'Link']
+    df1.to_csv('hacktoberfest_events.csv')
```
Comment (chavarera) [inline Scripts/Web_Scrappers/Hacktoberfest_Events/hacktoberfest_events.py:38]: write functions to perform single task

create a separate function to csv file

**Review by chavarera — CHANGES_REQUESTED** [review]

**Review by chavarera — APPROVED** [review]
good work @AdityaJ7 no new changes required

----------------------------
Make a common utility file to handle network request or handling specific operation so we can use that same utility functions in upcoming coding projects.

**Comment by AdityaJ7** [comment]
> good work @AdityaJ7 no new changes required
> 
> Make a common utility file to handle network request or handling specific operation so we can use that same utility functions in upcoming coding projects.

Ok!

**Pull request merged**